.github/workflows/workflow.yml

```diff
@@ -4,26 +4,20 @@ name: xrpl4j-CI
 on:
   push:
   pull_request:
-    types: [assigned]
+    types: [ assigned ]
 jobs:
   build_java8:
     runs-on: ubuntu-20.04
     steps:
       # Checks-out the repository under $GITHUB_WORKSPACE
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       # Set up Java 8
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK 8
+        uses: actions/setup-java@v4
         with:
-          java-version: 1.8
-      - name: Cache mvn dependencies
```

Comment:
Note that `actions/setup-java@v4` now includes automatic maven dependency caching, so this section is removed from all configurations.